wasm/grol_wasm.html

```diff
@@ -38,12 +39,46 @@
 </style>
 <script src="wasm_exec.js"></script>
 <script>
-    function debounce(func) {
+    /**
+     * A singular task ES6++ mutex
+     * inspired by this https://blog.jcoglan.com/2016/07/12/mutexes-and-javascript/
```

Comment:
I was asking on discord about JS mutexes... before being sent the debounce route

does that queue mean if I press the button 10 times (or hit return 10 times in text entry), it'll queue 10 runs? I'm not sure that's better than throwing them away ?